### ISSUE TITLE
[Docs content] guidance images

### ISSUE THREAD

**Issue by RobTobias123** [centrica-engineering/nucleus-docs#263]
### Outcome

All content-guidance images for components transferred to Astro Docs:
`/src/assets/components/ns-xxxx/guidance[-type][-state].webp`

### Scope

- [x] Add existing optimised webps to relevant images folder
- [x] Check and update any that require it
- [x] Check naming convention is applicable

**Comment by RobTobias123** [comment]
Relative to #245

**Comment by andij** [comment]
At this point time we are going keep the component guidance as simple to 'bring over' as possible.

There have been a number of ideas related to improving the guidance for each component type, most of which are captured in ticket #253.

Additionally there are a few mock-ups noted here: [#280#issuecomment-1870112833 ](https://github.com/centrica-engineering/nucleus-docs/issues/280#issuecomment-1870112833)

**Comment by RobTobias123** [comment]
All current guidance images have been added to PR #317, including experiences (nsx) and deprecated components. Note that there are no current images for a couple of the components such as ns-tab and ns-header (although there is one for nsx-header), and some were not uploaded as not required due to being replaced with sufficient HTML examples such as ns-inputter. I consider these to be up to date with the changes made to components for Nucleus v3.0 (eg. some ns-landmark size and spacing changed) and I have also added the new component of ns-action-link.
